tests/pyccel/scripts/return_numpy_arrays.py

```diff
@@ -1,15 +1,23 @@
+# pylint: disable=missing-function-docstring, disable=unused-variable, missing-module-docstring/
 import numpy as np
+from pyccel.decorators import types
 
 def single_return():
     a = np.array([1,2,3,4])
     return a
 
-def multi_returns():
-    x = single_return()
-    z = np.array([1,2,3,4])
-    return x, z
+@types('bool', 'bool')
+@types('int', 'int')
+@types('float', 'float')
+@types('complex', 'complex')
+def f(a, b):
+    c = np.array([a,b])
+    return c
 
 a = single_return()
-b,c = multi_returns()
+b = f(1, 3)
+c = f(1., 3.)
+d = f(False, True)
+e = f(1+2j, 3+4j)
 
-print(a, b, c)
+print(a, b, c, d, e)
```

Comment:
These statements need to be in a main for a program to be generated. I.e. this section should begin with:
```
if __name__ == '__main__':
```